src/pages/subreddit/Subreddit.js

```diff
@@ -1,40 +1,48 @@
 import React, { useEffect, useState } from 'react';
 import './Subreddit.css';
 import axios from 'axios';
-import {Link, useHistory} from "react-router-dom";
+import {useParams} from "react-router-dom";
 
 
 function Subreddit() {
-    const history = useHistory();
-    const [subreddit, setSubreddit] = useState('');
+    const [subreddit, setSubreddit] = useState({});
```

Comment:
Hier moet je dan die default waarde aanpassen
```suggestion
    const [subreddit, setSubreddit] = useState(undefined);
```